src/run.ts

```diff
@@ -84,7 +53,7 @@ export var walkSync = function (dir, filelist, fileToFind) {
 };
 
 export async function downloadHelm(version: string): Promise<string> {
-    if (!version) { version = await getStableHelmVersion(); }
+    if (!version) { version = await getLatestHelmVersion(); }
```

Comment:
We can just throw an error if helm version is undefined here. That makes it easier to maintain because getLatestHelm will only be called once and the flow will be more obvious. There should never be a point in the code where this is called with an undefined version since the version input is required.